gettsim/data/soz_vers_beitr.yaml

```diff
@@ -1,1192 +1,749 @@
+  2011-01-01:
+    deviation_from: previous
+    ges_krankv:
+      ag: 0.073
+      an: 0.082
+    arbeitsl_v: 0.015
+  2012-01-01:
+    deviation_from: previous
+    rentenv: 0.098
+    note: Quelle http://www.bmas.de/DE/Service/Presse/Pressemitteilungen/rentenbericht-16-11-2011.html
+  2013-01-01:
+    deviation_from: previous
+    rentenv: 0.0945
+    pflegev:
+      standard: 0.01025
+  2015-01-01:
+    ges_krankv:
+      ag: 0.073
+      an: 0.082
+    pflegev:
+      standard: 0.01175
+      zusatz_kinderlos: 0.0025
+    arbeitsl_v: 0.015
+    rentenv: 0.0935
+  2016-01-01:
+    deviation_from: previous
+    note: Bekanntmachung Bundesanzeiger 30.12.2015 und  http://www.bmg.bund.de/ministerium/meldungen/2015/gkv-zusatzbeitrag.html
+    ges_krankv:
+      an: 0.084
+  2017-01-01:
+    deviation_from: previous
+    pflegev:
+      standard: 0.01275
+  2018-01-01:
+    deviation_from: previous
+    ges_krankv:
+      an: 0.083
+    rentenv: 0.093
+  2019-01-01:
+    note: Wieder-Einführung der Parität. Art. 2 G. v. 11.12.2018, BGBl. I S. 2387. Durchschnittlicher zusatzbeitragssatz 0,9%
+    ges_krankv:
+      ag: 0.0775
+      an: 0.0775
+    pflegev:
+      standard: 0.01525
+      zusatz_kinderlos: 0.0025
+    arbeitsl_v: 0.013
+    rentenv: 0.093
+  2020-01-01:
+    note: Durchschnittlicher Zusatzbeitragssatz 1,1%, https://www.bundesgesundheitsministerium.de/presse/pressemitteilungen/2019/4-quartal/gkv-zusatzbeitragssatz.html
+    deviation_from: previous
+    ges_krankv:
+      ag: 0.0785
+      an: 0.0785
+    arbeitsl_v: 0.012
+
+beitr_bemess_grenze:
   name:
-    de: Beitragssatz zur Arbeitslosenversicherung (AN / AG)
-    en: Contribution rate to statutory unemployment insurance
+    de: Beitragsbemessungsgrenzen für die Sozialversihcerungsbeiträge
+    en: Income Threshold for social insurance.
   description:
-    de: Höhe § 341 (2) SGB III. Hälfte des gesetzlichen Werts.
+    de: Die Beitragsbemessungsgrenze für Kranken- und Pflegeversicherung ist die gleiche.
     en:
-  values:
-    1984-01-01:
-      value: 0.023
-      note:
-    1985-01-01:
-      value: 0.022
-      note:
-    1986-01-01:
-      value: 0.0205
-      note:
-    1987-01-01:
-      value: 0.0215
-      note: .
-    1992-01-01:
-      value: 0.0315
-      note: .
-    1993-01-01:
-      value: 0.0325
-      note: .
-    2007-01-01:
-      value: 0.021
-      note:
-    2008-01-01:
-      value: 0.0165
-      note:
-    2009-01-01:
-      value: 0.014
-      note:
-    2011-01-01:
-      value: 0.015
-      note:
-    2019-01-01:
-      value: 0.013
-      note: geändert durch BGBl. I S.2651
-    2020-01-01:
-      value: 0.012
-      note: https://www.bundesregierung.de/breg-de/aktuelles/beitrag-alv-1693634
-grvbs:
-  name:
-    de: Beitragssatz zur Gesetzlichen Rentenversicherung (AN / AG)
-    en: Contribution rate to statutory pension insurance (for both employers and employees)
-  description:
-    de:
+  note:
+    de: Vor 1990 Ost und West sind gleich.
```

Comment:
does not really makes sense. There was no "Ost" before 1990.

Reply:
Removed Ost values before 1990. They are now just missing, so if there is a data point which is recorded before 1990 and has a misspecified east/west Germany identifier, there will be an error now.